sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/UpdateDigitalTwinRequestOptions.java

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.digitaltwins.core;
+
+import java.util.List;
+
+/**
+ * Optional settings that are specific to calls to {@link DigitalTwinsClient#updateDigitalTwin(String, List)} and its overloads.
+ */
+public class UpdateDigitalTwinRequestOptions extends RequestOptions {
+    // This class exists to be added to later if the deleteDigitalTwin APIs get a new optional parameter in later service
```

Comment:
Copy/paste error in this comment. this is not used by the deleteDigitalTwin API

Reply:
done done